### PR TITLE
11 other sections on the index page

### PR DESCRIPTION
**Pull request by jasonzhwang**
AC1: implement Other Sections Page in desktop width:1920px:
Screenshot 2023-06-11 at 9 35 40 am

AC2:
refactor Button component and styles from tailwind to scss introduced from Taotify:
type1: pass color & size props to create normal button
Screenshot 2023-06-11 at 9 38 40 am
type2: pass color & size props & Image & WidthIncreasing Props to create Image Bouncing button
Screenshot 2023-06-11 at 9 44 47 am
(!please notice width increasing is added because sometimes the picture size needed to be same as button's, but sometimes not)

AC3: Image can bounce when hovering
Screenshot 2023-06-11 at 9 48 12 am
Screenshot 2023-06-11 at 9 48 23 am

### CODE REVIEW

**Comment by tatoMa** [comment]
There is a noticeable difference between UI design and this PR.
image

**Comment by tatoMa** [comment]
I know you export all styles from Figma and it is a quick way to do the job, but something still needs refactoring. Eg. hierarchy, font size, and default font family for heading or regular text.

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1,17 +1,59 @@
...
       </section>
       <section>
-        <h2>Learn a language in a playful way</h2>
-      </section>
-      <section>
-        <h2>Increase your vocabulary</h2>
-      </section>
-      <section>
-        <h2>Watch your progress everyday</h2>
+        <section className="languageSectionContainer">
```
Comment (LULIU-9) [inline src/pages/index.tsx:11]: I recommend refactoring the sections into separate components for better code organization and reusability. Additionally, the vocabularySectionContainer and statisticsSectionContainer appear to have the same structure. Consider refactoring them into a single component to reduce duplication and improve maintainability.

Comment (jasonzhwang) [reply src/pages/index.tsx]: I have already extracted sections from main page to the OtherSections component.

Regard to the current code, I did think about that not even vocabularySectionContainer and statisticsSectionContainer, but also VocabularySectionContainer, which three other sections can form as one component.

But I'd like to wait to the next refactoring sprint, as the possible responsive may be added on, which may potential increase the props number passing to the component.

File: src/styles/global.scss [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+/***     HomePage      ***/
+.languageSectionContainer {
+  @include sectionGridLayout;
+  gap: 120px;
+  padding: 77.5px 50px;
+  img,
+  div:first-child {
+    @include verticalCentered;
+  }
```
Comment (Double-Twelve) [inline src/styles/global.scss:34]: you should use `first-of-type` instead of `first-child`

**Comment by Double-Twelve** [comment]
please write tests

**Comment by jasonzhwang** [comment]
> please write tests

Hi Double, I am still learning testing and I get the commit from Tao that I will open up another ticket for testing after the 
 current ticket. Thanks.

**Review by Double-Twelve — APPROVED** [review]

**Pull request merged**